code-scanning/roxctl.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# This workflow integrates roxctl with GitHub's Code Scanning
+# feature.
+#
+# roxctl is the CLI of StackRox, a container security platform.
+# roxctl will scan container images for vulnerabilities.
+#
+# See https://www.stackrox.io/docs/ for more on StackRox.
+#
+# See https://github.com/stackrox/roxctl-installer-action for
+# more on the roxctl-installer action and its parameters.
+
+name: Roxctl
```

Comment:
`Roxctl` or `roxctl`? I think we usually use the latter, right?

Reply:
Initially thought the naming pattern required upper-case but it doesn't so changed it now to lower-case.